qiskit/backends/local/qasm_simulator_cpp.py

```diff
@@ -51,19 +52,18 @@ class QasmSimulatorCpp(BaseBackend):
     """C++ quantum circuit simulator with realistic noise"""
 
     DEFAULT_CONFIGURATION = {
-        'name': 'local_qiskit_simulator',
+        'name': 'local_qasm_simulator_cpp',
         'url': 'https://github.com/QISKit/qiskit-sdk-py/src/qasm-simulator-cpp',
         'simulator': True,
         'local': True,
         'description': 'A C++ realistic noise simulator for qobj files',
         'coupling_map': 'all-to-all',
-        "basis_gates": 'u1,u2,u3,cx,id,x,y,z,h,s,sdg,t,tdg,rzz,' +
+        "basis_gates": 'u1,u2,u3,cx,cz,id,x,y,z,h,s,sdg,t,tdg,rzz,' +
```

Comment:
`u0` should be included as a basis gate for the cpp simulator

Reply:
good catch.